src/main/java/ch/njol/skript/conditions/CondChance.java

```diff
@@ -40,8 +40,8 @@
 		"Valid values are between 0% and 100%, or if the percent sign is omitted between 0 and 1."})
 @Examples({"chance of 50%:",
```

Comment:
```suggestion
@Examples({"if chance of 50%:",
```

Reply:
There are conditions that just look bad wih the if, this one being the case.

Reply:
Yeah, I agree with that